app/src/main/res/drawable/drawer_background.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
As a general rule, we create our designs for high resolutions, so all images should be placed in xxhdpi folder. Only xml drawable files should be placed in the default drawable folder